run.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
+
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Parameters
+WORKLOAD="${1}"
+PROJECT="${2}"
+BUCKET_NAME="${3}"
+BIGQUERY_DATASET="${4}"
+API="${5}"
+OBJECT_SIZE="${6}"
+SAMPLES="${7}"
+WORKERS="${8}"
```

Comment:
maybe check `$#` to make sure the right number of parameters was passed in?  Maybe consider using `getopt`:


https://github.com/googleapis/google-cloud-cpp/blob/e675f70d673e9e6d1cbb537b391d9222d6e5fbb1/ci/cloudbuild/convert-to-branch-triggers.sh#L71

Reply:
+1 for using `getopts`, assuming this will actually be run by humans at some point. https://gist.github.com/reikind/4072830